tests/dataframes_test.py

```diff
@@ -199,7 +285,7 @@ def test_run_query(self, mock_aggregate, metrics, public_keys):
         columns = dataframes.Columns("privacy_key", "group_key", value_column)
         metrics_dict = dict([(m, m.name) for m in metrics])
         min_value = max_value = None
-        if pipeline_dp.Metrics.SUM in metrics:
+        if pipeline_dp.Metrics.SUM in metrics or pipeline_dp.Metrics.MEAN in metrics:
```

Comment:
mhh, I personally don't very like when we start writing a lot of "if" conditions specific to parameters of a test, it makes the test complicated, I think in many cases copy pasting is better, improves test readability, but up to you. We can say that it is more or less "integration" ("big") test and it is ok for it to have complicated arrange and assert sections.